Khan/wonder-stuff
# Enable aphrodite-add-style-variable-name rule in eslint-plugin-khan demo project

## Summary:
Now that the `eslint-plugin-khan` package is released with the new `aphrodite-add-style-variable-name` rule, we can update the demo project to test that the rule works as expected

Issue: FEI-5952

## Test plan:
Linting error
- Run `yarn lint` in `packages/eslint-plugin-khan/demo`
- Notice there is an error for the `aphrodite-add-style-variable-name` rule

<img width="1056" alt="Screenshot 2024-12-13 at 4 19 12 PM" src="https://github.com/user-attachments/assets/80ce04ae-2526-4c31-bc85-d8d4bc3720de" />

Hints in vs code
- Open `packages/eslint-plugin-khan/demo` in vs code (the demo project needs to be opened separately)
- Open the `foo.tsx` file
- Notice the linting error on the variable name when using addStyle

<img width="1313" alt="Screenshot 2024-12-13 at 4 22 04 PM" src="https://github.com/user-attachments/assets/55faf91f-cd4d-40c4-89dc-6b55ee00e10b" />
